### DISCUSSION TITLE
How do i align text to an image in mantine?

### DISCUSSION THREAD

**Discussion by starxyn** [mantinedev/mantine#6851] [Q&A]
I have an image inside a grid and wanted to place a text beneath the image and center to it. How do i do that?
`
<Grid.Col span={{ base: 12, xs: 2}}>
  <Image>
  <Text>content</Text>
</Grid.Col>
<Grid.Col span={{ base: 12, xs: 10}}>
</Grid.Col>
`

**Comment by Kenzo-Wada** [comment]
Is this what you want to do?
https://mantine.dev/core/background-image/#usage
